src/components/CustomInput.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <label >
+        <span
+            v-if="label"
+            class="base__label">
+            {{ label }}
+        </span>
+        <input 
+            :value="value"
+            @input="$emit('input', $event.target.value)"
+            :placeholder="placeholder"
+            :type="type"
+            class="base__input">
+            
+    </label>
+</template>
+
+<script>
+    export default {
+        props: {
+            value: {
+                type: String,
+                default: ''
+            },
+            label: {
+                type: String,
+                default: ''
+            },
+            placeholder: {
+                type: String,
+                default: ''
+            },
+            type: {
+                type: String,
+                default: 'text'
+            }
+        }
+    }
+</script>
+
+<style lang="scss">
+.base {
+    &__label {
```

Comment:
If you are using BEM, it is a good practice to have class names somehow related to component name. In other case you should at least make those styles `scoped`. If you won't follow any of those two conventions it is possible that other component will have the same class name at some point (e.g. `.base__label`) and this conflict may lead to unexpected behavior.